src/Build/Microsoft.Build.csproj

```diff
@@ -33,6 +33,7 @@
 
     <PackageReference Include="System.Collections.Immutable" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" />
+    <PackageReference Include="System.Memory" />
```

Comment:
This will require some Setup Magic, so that we have it available in VS setup.

Notes for that: #3947, https://github.com/microsoft/msbuild/pull/3648/files#diff-ebf6dde28bf300b10540e8b71b872c5d.

Let's sync on that and pair through it (maybe next week while I'm there?).